packages/suite/src/views/dashboard/components/PortfolioCard/components/Exception.tsx

```diff
@@ -53,7 +53,7 @@ interface CTA {
 
 interface ContainerProps {
     title: ComponentProps<typeof Translation>['id'];
-    description: ComponentProps<typeof Translation>['id'] | JSX.Element;
+    description?: ComponentProps<typeof Translation>['id'] | JSX.Element;
```

Comment:
`ComponentProps<typeof Translation>['id']` could be simplified to `TranslationKey`.